Lab5.cpp

```diff
@@ -0,0 +1,28 @@
+﻿#include <iostream> 
+#include <string>
+#include "Methods.h"
+using namespace std;
+#define ARRAY_SIZE 3    
+
+int main() {
+    Shop a;
+    int input;
+    float money;
+    while (1) {
```

Comment:
You use a magic number.

You should:
while (true) {

or

#define CYCLIC_SURVEY_OF_THE_BUYER true
...code...
do {
...code...
} while (CYCLIC_SURVEY_OF_THE_BUYER);